src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -81,14 +82,14 @@ Datasets containing the `Genetics` field in `dataset_description.json` or the
-| SampleOrigin                                                | REQUIRED. String. Describes from which tissue the genetic information was extracted. Values MUST be one of `blood`, `saliva`, `brain`, `csf`, `breast milk`, `bile`, `amniotic fluid`, `other biospecimen`.                                                                              |
-| TissueOrigin                                                | OPTIONAL. String. Describes the type of tissue analyzed for SampleOrigin `brain`. Values MUST be one of `gray matter`, `white matter`, `csf`, `meninges`, `macrovascular` or `microvascular`.                                                                                            |
-| BrainLocation                                               | OPTIONAL. String. Refers to the location in space of the `TissueOrigin`. Values may be an MNI coordinate, a label taken from the [Allen Brain Atlas][allen], or layer to refer to layer-specific gene expression, which can also tie up with laminar fMRI.                               |
-| CellType                                                    | OPTIONAL. String. Describes the type of cell analyzed. Values SHOULD come from the [cell ontology][ontology].                                                                                                                                                                            |
+| **Key name**       | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                              |
+| ------------------ | --------------------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| GeneticLevel       | REQUIRED              | [string][] or [array][] of [strings][] | Describes the level of analysis. Values MUST be one of `Genetic`, `Genomic`, `Epigenomic`, `Transcriptomic`, `Metabolomic`, or `Proteomic`.                                                                                                  |
+| AnalyticalApproach | OPTIONAL              | [string][] or [array][] of [strings][] | Methodology or methodologies used to analyse the GeneticLevel. Values MUST be taken from the [database of Genotypes and Phenotypes (dbGaP)][gapsolr] under /Study/Molecular Data Type (e.g., `SNP Genotypes (Array)` or `Methylation (CpG)`. |
```

Comment:
fair enough, but if okay with you I'd put that off to another PR, because the spec is full of `e.g.` and changing everything would blow up the PR --> those E.G.s I touched *here*, were touched for adding a trailing comma to be consistent.

Reply:
Sure.